vnet/bin/vnflows-monitor

```diff
@@ -464,57 +442,30 @@ def output_flows(outputflows)
     
     puts flow.to_pretty
     linecount = 1 + linecount
-    break if ( $opts.head > 0 ) && ( linecount > $opts.head )
+    break if ( $opts[:head] > 0 ) && ( linecount > $opts[:head] )
   }
   puts "-------run:#{$run_n}--iteration:#{$inputcount}-------"
   $count = $count + 1  
-  exit if ( $opts.maxcount > 0 ) && ( $count >= $opts.maxcount )
+  exit if ( $opts[:count] > 0 ) && ( $count >= $opts[:count] )
 end
 
 def do_sleep
-  sleep $opts.interval
+  sleep $opts[:time]
 end
 
 def update_options
   while $stdin.ready?
-    $opts.parse_options $stdin.readline
+    $stdin.readline
   end
 end
 
-if /help/.match(ARGV.to_s)
-  puts "
-c 0   -keep iterating
-c 1   -output one iteration and stop {default}
-c n   -output n iterations and stop
-
-d 1   -turn on diff mode
-d 0   -turn off diff mode {default}
-
-p 0   -full output
-p 1   -only show low 32bits of metadata value {default}
-
-t 2   -pause 2 seconds between iterations {default}
-
-r xxx -pull flows from previous run number xxx
-       {defaults to most recent run}
-
-i xxx -pull flows from iteration xxx
-
-d 1 i xxx yyy r zzz
-      -do diff between xxx and yyy of run zzz
-
-(Many of options above can be typed into stdin while running)
-"
-  exit
-end
-$opts.parse_options(ARGV.to_s)
-
-iters = $opts.iterationselect
-if iters.length != 0
-  if $opts.runselect == 0
+if $opts[:iteration]
+  put "iterations"
+  iters = $opts[:iteration]
+  if $opts[:run] == 0
     latest = latest_run_number
```

Comment:
To use the diff feature on archived data, you need to specify two iterations.  That is why $opts.iterationselect was an array.  See the example in the help "d 1 i xxx yyy r zzz".  Both xxx and yyy would get appended to the array.  If you want to break this feature (as the current pull request does), the code in the "iters.length" below can be removed.